src/ResourceManager/ContainerRegistry/ChangeLog.md

```diff
@@ -0,0 +1,30 @@
+<!--
+    Please leave this section at the top of the change log.
+
+    Changes for the current release should go under the section titled "Current Release", and should adhere to the following format:
+
+    ## Current Release
+    * Overview of change #1
+        - Additional information about change #1
+    * Overview of change #2
+        - Additional information about change #2
+        - Additional information about change #2
+    * Overview of change #3
+    * Overview of change #4
+        - Additional information about change #4
+
+    ## YYYY.MM.DD - Version X.Y.Z (Previous Release)
+    * Overview of change #1
+        - Additional information about change #1
+-->
+## Current Release
+
+## Version 1.0.0
```

Comment:
@djyou I suppose the header should say "Upcoming Release" instead of "Current Release". Regardless, this is the pattern that we have followed for every RP:

- Service makes change(s) to their cmdlet(s)
- Change log is updated (under the "Current Release" section) to update these changes
- During release, module versions are updated, and "Current Release" header is changed to reflect the released version, and a new "Current Release" header is created
- Customers can view that 1.0.0 of ContainerRegistry was just released and can view the notes about the changes under the "Version 1.0.0" section of the change log (or in the release notes on PowerShell Gallery)